OpenRA.Mods.Cnc/Traits/Minelayer.cs

```diff
@@ -268,7 +271,9 @@ public bool CanTarget(Actor self, Target target, List<Actor> othersAtTarget, ref
 				if (!self.World.Map.Contains(location))
 					return false;
 
-				cursor = "ability";
+				var minelayer = self.Trait<Minelayer>();
```

Comment:
Please pass `MineLayerInfo` into the `BeginMinefieldOrderTargeter` constructor instead of querying it all the time here.